File: C++/1035.cpp
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main(){

    int a, b, c, d;

    cin >> a >> b >> c >> d;

    if((a%2==0) && (b>c) && (c+d > a+b) && (d>a) && (c > 0) && (d > 0))
        cout << "Valores aceitos\n";
    else
        cout << "Valores nao aceitos\n";

}

File: C#/1035.cs
using System; 

class URI {

    static void Main(string[] args)        {
            
            string[] linha1 = Console.ReadLine().Split(' ');
            int a = int.Parse(linha1[0]);
            int b = int.Parse(linha1[1]);
            int c = int.Parse(linha1[2]);
            int d = int.Parse(linha1[3]);
            if ((b > c) && (d > a) && ((c + d) > (a + b)) && (c > 0) && (d > 0) && (a % 2 == 0))
                Console.WriteLine("Valores aceitos");
            else
                Console.WriteLine("Valores nao aceitos");
        }

}